pkgs/top-level/all-packages.nix

```diff
@@ -8276,6 +8276,10 @@ with pkgs;
 
   minidlna = callPackage ../tools/networking/minidlna { };
 
+  miniplayer = callPackage ../applications/audio/miniplayer {
+    python3Packages = python39Packages;
```

Comment:
I just double checked and couldn't find a reason to use python 3.9 in the upstream source code.

Reply:
I've removed that override. Strangely enough it didn't work for 3.10 [before](https://github.com/NixOS/nixpkgs/pull/190380#issuecomment-1241692956).